components/Hero/StoryHeroMvp.tsx

```diff
@@ -72,46 +65,6 @@ export const StoryHeroMvp = ({
     year: 'numeric',
   });
 
-  const cropSize = styles.imageCrops[aspectRatio];
-  const cropWidth = parseInt(cropSize?.split('x')[0], 10);
-  const cropHeight = parseInt(cropSize?.split('x')[1], 10);
-  const mobileCropSize = styles.mobileImageCrops[mobileAspectRatio];
-  const mobileCropWidth = parseInt(mobileCropSize?.split('x')[0], 10);
-  const mobileCropHeight = parseInt(mobileCropSize?.split('x')[1], 10);
-
-  const renderOneImage = !!filename && !mobileFilename && mobileAspectRatio === aspectRatio;
-  // Render 2 different images if there is both a hero image or a mobile image
-  // Or when there is no mobile image, but the mobile aspect ratio is different from the desktop aspect ratio
-  const renderTwoImages = (!!filename && !!mobileFilename) ||
-    (!!filename && !mobileFilename && mobileAspectRatio !== aspectRatio);
-
-  // TODO: add srcset later in GIVCAMP-71
-  // This one will always be rendered when there's a hero image
-  const RenderedDesktopImage = (
-    <img
-      alt={alt || ''}
-      loading="eager"
-      width={cropWidth}
-      height={cropHeight}
-      src={getProcessedImage(filename, cropSize, focus)}
-      sizes={isVerticalHero ? `${renderOneImage ? '(max-width: 991px) 100vw, 50vw}' : '50vw'}` : '100vw'}
-      className={styles.image(renderTwoImages)}
-    />
-  );
-
-  // This one will only be rendered when the conditions in renderTwoImages are met
-  const RenderedMobileImage = renderTwoImages && (
-    <img
-      alt={alt || ''}
-      loading="eager"
-      width={mobileCropWidth}
-      height={mobileCropHeight}
-      src={getProcessedImage(mobileFilename || filename, mobileCropSize, mobileFocus || focus)}
-      sizes="100vw"
-      className={styles.mobileImage}
-    />
-  );
-
```

Comment:
I copied this code in the original SbStory (we're using SbStoryMvp), we no longer need aspect ratio and mobile image so these are outdated.